### ISSUE TITLE
Tag documents are generated with empty ExtractedText fields

### ISSUE THREAD

**Issue by ghost** [socs-dev-env/dosocs2#30]
Example:

```
LicenseID: LicenseRef-Python
LicenseName: Python
ExtractedText:
LicenseCrossReference:
LicenseComment: <text>found by nomos</text>
```

This is technically invalid. The `ExtractedText` field should at least have the `<text>` tags.

UPDATE 6/17/2015: Even adding the <text> tags is insufficient... the document still fails validation because this field needs to actually have _something in it_; it needs to be populated with license text. This is a harder problem.

**Comment by ghost** [comment]
Fixed in dev.

**Comment by ghost** [comment]
#30 is fixed with commit  `da9b964`
